docs/apis/subsystems/files/index.md

```diff
@@ -215,7 +215,8 @@ function mod_myplugin_pluginfile(
     }
 
     // We can now send the file back to the browser - in this case with a cache lifetime of 1 day and no filtering.
-    send_stored_file($file, DAY_SECS, 0, $forcedownload, $options);
+    $daySecs = 60*60*24;
```

Comment:
This is incorrect. THe existing line is correct. Please use the `DAY_SECS` constant rather than creating new variables (with incorrect coding naming conventinos)

Reply:
Hm interesting i can't access this constant.

Reply:
Ah - that's because it is DAYSECS and not DAY_SECS :)
Could you update your patch to fix the naming and then we hsould be all good :)